### DISCUSSION TITLE
Retronas with Stellar for Xbox

### DISCUSSION THREAD

**Discussion by COIh0rp** [retronas/discussions#126] [General]
Is it possible to get some advice on what is the best way to get retronas working with Stellar for xbox? The recent firmware update for stellar has enabled netiso loading, but I have been having trouble getting it working with retronas. I've also looked the xbox manager in cockpit, but am getting failed to read config file when I try to update anything. Thanks!

**Comment by sairuk** [comment]
xbox manager is a separate project to retronas but i'll take a look at it.

as for project stella netiso support, i don't have one but reading the doco it should work with any samba share you install to retronas providing it is setup correctly. you can always test the share by accessing it from another machine.

**Reply by sairuk** [reply]
xbox manager should be fixed now, was a bug with limited access readability of the config file, reinstall and test and a dependency was failing to install on deb12
